content/ops/creating-a-local-dev-environment-in-Virtual-Box.md

```diff
@@ -196,8 +196,6 @@ Secure copy the cflinuxfs2 container to the CF VM.
 
 Connect using the Cloud Foundry CLI.
 
-	# Run this from inside your Vagrant Bosh Lite instance
-	vagrant ssh
```

Comment:
Was this just a mistake before? Or do you no longer run the commands from within the Vagrant instance?